File: openapi/code_samples/go/api_v1_organizations_{organization_id}_users/post.go
newUser := &usersv1.CreateUser{
    Email:      "user@example.com",
    ExternalId: "ext_12345a67b89c",
    Metadata: map[string]string{
        "department": "engineering",
        "location":   "nyc-office",
    },
    UserProfile: &usersv1.CreateUserProfile{
        FirstName: "John",
        LastName:  "Doe",
    },
}
cuResp, 
  err := scalekitClient.User().CreateUserAndMembership(ctx, "org_123",
newUser, false) if err != nil { /* handle error */ }

File: openapi/code_samples/javascript/api_v1_organizations_{organization_id}_users/post.js
const { user } = await scalekit.user.createUserAndMembership('org_123', {
  email: 'user@example.com',
  externalId: 'ext_12345a67b89c',
  metadata: { department: 'engineering', location: 'nyc-office' },
  userProfile: {
    firstName: 'John',
    lastName: 'Doe',
  },
})
